modules/data.atmosphere/inst/integrationTests/test.download.AmerifluxLBL.R

```diff
@@ -0,0 +1,31 @@
+# putting logger to debug mode
+PEcAn.logger::logger.setUseConsole(TRUE, FALSE)
+on.exit(PEcAn.logger::logger.setUseConsole(TRUE, TRUE), add = TRUE)
```

Comment:
Not sure that `on.exit()` works as expected outside of a function.  It will trigger only when an enclosing function environment is exited I think so this may not happen until after all tests are run if run with `test()`